### PR TITLE
Fix:Issue #826-Disbursement datepicker

### PR DESCRIPTION
**Pull request by AkshGautam**
Fix: #826
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
@therajanmaurya @tarun0  MFDatePicker class could be made more user-friendly by improving the new Instance method in it.Prior to this addition it was easier for someone to use date picker with maxdate but using min date would require a way around. I had to make another method to do this. The behaviour could be made more general by improving the new instance method but that would require changing code wherever MFdatepicker is used in code,That could be done after discussion.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java [hunk 1]
```diff
@@ -21,6 +21,10 @@
  */
 public class MFDatePicker extends DialogFragment implements DatePickerDialog.OnDateSetListener {
 
+    private static final String DATE_KEY = "date_present";
+    private static final int set_max_date = 0;
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java]: Don't use 0 and 1. Either use Strings in Caps like `FOR_MIN_DATE` or use ENUM. 0 and 1 isn't easy to comprehend for the new user.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java [hunk 2]
```diff
@@ -53,19 +57,37 @@ public static MFDatePicker newInsance(Fragment fragment) {
         return mfDatePicker;
     }
 
+    public static MFDatePicker newInstance(Fragment fragment, Integer id) {
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java]: Have you tested that it doesn't break anything else?

**Comment by droidchef** [comment]
Requested changes aren't submitted. Please reopen the PR when you've answered the comments from the reviewers.

**Pull request closed without merging**